src/helix-ui/styles/components/forms/asterisk.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+  <path fill="#D32F2F" d="M5.658 8.941L0 7.33l1.063-3.276 5.604 2.263L6.306 0h3.532l-.361 6.443 5.46-2.226L16 7.529 10.234 9.14l3.784 4.815L11.153 16l-3.279-5.285-3.243 5.104-2.865-1.955z"/>
```

Comment:
We have to define the `fill` attribute explicitly, because we have no way to modify the color with CSS, given that we embed the SVG as a Base64 data URI.